### PR TITLE
Initial work

### PR DESCRIPTION
**Pull request by Nava2**
This is a PoC showing the backend's output displayed in a simple table. This table has back links to Umple Online and the Umple file generated. I need to open an issue against the back end to keep the original files in the output directory as well so this can be displayed. 

Data has also been limited to a small amount due to trying not to code bloat the repository. In the future, a separate repository might be used to keep all content in one place, or stored on the server. The file path currently is hardcoded and will change.

### CODE REVIEW

**Comment by Nava2** [comment]
I really recommend just looking at the `home.html.php` since that's all I changed. I realized I need to add the file output for where the original content is saved (I forgot to add it as output in the backend..). The umpleonline links will work with a non-localhost server. :)

File: app/models/DiagramType.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<?php
+/*PLEASE DO NOT EDIT THIS CODE*/
```
Comment (aforward) [inline app/models/DiagramType.php:2]: Where are the underlying ump files?  If they were borrowed from the umplifier code, I would consider that coincidental duplication and the raw sources (aka the ump files) should be included in this project.  If there are genuine similarities that are worth keeping, then we can revisit that.

**Comment by aforward** [comment]
I don't see the "public/*.php" files, they are coming?

**Comment by aforward** [comment]
"Sorry, we could not display the entire diff because it was too big."

At least github said sorry.  The files are there, just not in the summary

**Comment by aforward** [comment]
I might suggest creating a separate PR with just simpletest to reduce the changeset on this one

**Comment by Nava2** [comment]
I rebased all of the changes, now the PR is much more manageable to read. master now has the original code before I started working against it. This PR just has my modified changes.

**Comment by Nava2** [comment]
Opened umple-ucosp/umple.sample.downloader#5 to address the lack of "file to import" being present (i.e. the original ECore in this simple example).

**Pull request merged**